packages/xo-server/src/api/vm.mjs

```diff
@@ -1358,8 +1358,8 @@ async function import_({ data, sr, type = 'xva', url }) {
     if (type !== 'xva') {
       throw invalidParameters('URL import is only compatible with XVA')
     }
-
-    const ref = await xapi.VM_import(await hrp(url), sr._xapiRef)
+    const timeout = this.config.get('jsonrpc-api').xvaImportFromUrlTimeout ?? 6e3
```

Comment:
I was expecting this to parse the duration but it is automatic?
We don't need something like this?
```suggestion
    const timeout = 6e3
    try {
       const timeout = this.config.getDuration('jsonrpc-api.xvaImportFromUrlTimeout')
    } catch(e) {
    }
```